spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe QueueItem do
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+  it do
+    should validate_uniqueness_of(:video_id).
+      scoped_to(:user_id)
+  end
+
+  describe '#video_title' do
```

Comment:
You can further refactor this with shoulda matchers. There is a `delegate_method` matcher - https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb